tools/bench/functional/permutor.py

```
@@ -0,0 +1,103 @@
+MAKE_TAR_FILE = True
+log_dir = Path(".log")
+local_dir = Path(".phare")
+out_dir = Path(f".phare_bench/{datetime_now()}")
+monitoring_options = MonitoringOptions(interval=5, rank_modulo=1)
+
+
+def sort_summaries(path):
+    import re
+
+    with open(path) as f:
+        lines = [l.rstrip("\n") for l in f if l.strip()]
```

Ruff (0.15.15)

[error] Ambiguous variable name: `l`

(E741)